tests/testthat/test_model_output.R

```diff
@@ -189,56 +189,6 @@ test_that("land area doesn't change over time", {
 
 })
 
-
-test_that("land cover matches reference values", {
```

Comment:
Wasn't this testing that when we ran a configuration where we knew the answer, we actually got the answer we were expecting?  If so, is there an equivalent test we can do to verify correctness?

Reply:
Yes, but that test appeared in two different test files. I removed this one and left it in the other.